cmd/tailscale/cli/set.go

```diff
@@ -82,6 +84,7 @@ func newSetFlagSet(goos string, setArgs *setArgsT) *flag.FlagSet {
 	setf.BoolVar(&setArgs.updateApply, "auto-update", false, "automatically update to the latest available version")
 	setf.BoolVar(&setArgs.postureChecking, "posture-checking", false, hidden+"allow management plane to gather device posture information")
 	setf.BoolVar(&setArgs.runWebClient, "webclient", false, "expose the web interface for managing this node over Tailscale at port 5252")
+	setf.StringVar(&setArgs.relayServerPort, "relay-server-port", "", hidden+"UDP port number (0 will pick a random unused port) to act as a relay server on between Tailscale nodes, or empty string to disable relay server functionality")
```

Comment:
in between?

Reply:
port on what interface(s)? just the Tailscale IP(s)?

Reply:
All interfaces (0.0.0.0 and ::) as this is an underlay relay.

How does https://github.com/tailscale/tailscale/pull/15594/commits/411280aa9c55d537072a58e5808fb8b403bd7661 sound?

Trying to fit relay server vs relay client relationship into the flag description ends up being a bit of a mouthful.